draft-ietf-tls-esni.md

```diff
@@ -430,6 +430,37 @@ the ClientHelloOuter. The server then computes the digest of the reconstructed
 ClientHelloInner. If the digest does not equal OuterExtensions.inner_digest,
 then the server MUST abort the connection with a "decrypt_error" alert.
 
+# The "handshake_padding" Extension {#handshake-padding}
```

Comment:
This should go in IANA Considerations, with a suitable TLS 1.3 column (all the messages).

Reply:
Good catch, thanks!